java/ql/lib/config/semmlecode.dbscheme

```diff
@@ -264,18 +307,31 @@ numlines(
   int num_comment: int ref
 );
 
+/**
+ * Associates the filepath `name` with the ID `id`.
+ */
 files(
   unique int id: @file,
   string name: string ref
 );
 
+/**
+ * Associates the filepath `name` with the ID `id`.
```

Comment:
filepath or folderpath?

Reply:
Ta; I've rewritten both files and folders docs to be clearer